astropy/tests/helper.py

```diff
@@ -463,10 +463,15 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
     This is a :class:`~astropy.units.Quantity`-aware version of
     :func:`numpy.testing.assert_allclose`.
     """
-    assert _quantity_allclose(actual, desired, rtol=rtol, atol=atol, **kwargs)
+    import numpy as np
```

Comment:
is there a particular reason to keep the numpy import here? I see that it was here originally, so merely just wonder to learn.

Reply:
I'm not sure why. I thought maybe it is related to not wanting `numpy` to be a hard dependency in this file for build or something. @mhvk ?